modules/central-services-public-config.adoc

```diff
@@ -40,6 +40,19 @@ Image declares the configuration to set up the main registry, which the Helm cha
 Either use a hostname, such as `registry.redhat.io`, or a remote registry hostname, such as `us.gcr.io/stackrox-mirror`.
 |===
 
+[id="central-services-public-configuration-file-config-as-code_{context}"]
+== Policy as code
+
+Policy as code provides a way to configure {product-title-short} to work with a continuous delivery tool such as Argo CD to track, manage, and apply policies that you have authored locally. You can also use the API to configure connections to your own GitOps repository such as GitHub.
+
+|===
+| Parameter | Description
+
+| `configAsCode.enabled`
+| Set to `true` to enable the policy as code feature.
```

Comment:
The default value is `true`, so would it make sense to invert the wording here?  "Set to `false` to disable the policy as code feature."